Notebooks/DC2_Postage_Stamps.ipynb

```diff
@@ -4,7 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## DC2: Generate Postage Stamps for set of RA, Dec coordinates\n",
+    "# DC2: Generate Postage Stamps for set of RA, Dec coordinates\n",
+    "<br>Owner: **Michael Wood-Vasey** ([@wmwv](https://github.com/LSSTDESC/DC2_Repo/issues/new?body=@wmwv))\n",
```

Comment:
I think we need an owner and a date for all notebook, to help guard against staleness.